bson/binary.py

```diff
@@ -364,4 +364,7 @@ def __ne__(self, other: Any) -> bool:
         return not self == other
 
     def __repr__(self) -> str:
-        return f"Binary({bytes.__repr__(self)}, {self.__subtype})"
+        if self.__subtype == SENSITIVE_SUBTYPE:
+            return f"<Binary ### {self.__subtype}>"
```

Comment:
For consistency, can we format it like `f"Binary('<redacted>', {self._subtype))`>

Reply:
Okay will do.